bitacross-worker/README.md

```diff
@@ -1 +1,200 @@
+    // e.g. unisat-wallet: https://docs.unisat.io/dev/unisat-developer-service/unisat-wallet#getpublickey
+    #[codec(index = 5)]
+    Bitcoin(Address33),
+}
+
+pub enum LitentryMultiSignature {
+    /// An Ed25519 signature.
+    #[codec(index = 0)]
+    Ed25519(ed25519::Signature),
+    /// An Sr25519 signature.
+    #[codec(index = 1)]
+    Sr25519(sr25519::Signature),
+    /// An ECDSA/SECP256k1 signature.
+    #[codec(index = 2)]
+    Ecdsa(ecdsa::Signature),
+    /// An ECDSA/keccak256 signature. An Ethereum signature. hash message with keccak256
+    #[codec(index = 3)]
+    Ethereum(EthereumSignature),
+    /// Same as above, but the payload bytes are prepended with a readable prefix and `0x`
+    #[codec(index = 4)]
+    EthereumPrettified(EthereumSignature),
+    /// Bitcoin signed message, a hex-encoded string of original &[u8] message, without `0x` prefix
+    #[codec(index = 5)]
+    Bitcoin(BitcoinSignature),
+    /// Same as above, but the payload bytes are prepended with a readable prefix and `0x`
+    #[codec(index = 6)]
+    BitcoinPrettified(BitcoinSignature),
+}
+
+#[derive(Encode, Decode, MaxEncodedLen, TypeInfo, PartialEq, Eq, Clone, Debug)]
+pub struct EthereumSignature(pub [u8; 65]);
+
+#[derive(Encode, Decode, MaxEncodedLen, TypeInfo, PartialEq, Eq, Clone, Debug)]
+pub struct BitcoinSignature(pub [u8; 65]);
+
+#[derive(
+Encode, Decode, Copy, Clone, Default, PartialEq, Eq, TypeInfo, MaxEncodedLen, Ord, PartialOrd,
+)]
+pub struct Address20([u8; 20]);
+
+#[derive(
+Encode, Decode, Copy, Clone, Default, PartialEq, Eq, TypeInfo, MaxEncodedLen, Ord, PartialOrd,
+)]
+pub struct Address32([u8; 32]);
+
+#[derive(Encode, Decode, Copy, Clone, PartialEq, Eq, TypeInfo, MaxEncodedLen, PartialOrd, Ord)]
+pub struct Address33([u8; 33]);
+
+#[derive(Debug, Default, Clone, Eq, PartialEq, Encode, Decode)]
+pub struct AesOutput {
+    pub ciphertext: Vec<u8>,
+    pub aad: Vec<u8>,
+    pub nonce: RequestAesKeyNonce, // IV
+}
+
+#[derive(Encode, Decode, Debug, Eq, PartialEq)]
+pub struct RpcReturnValue {
+    pub value: Vec<u8>,
+    pub do_watch: bool,
+    pub status: DirectRequestStatus,
+}
+
+#[derive(Debug, Clone, PartialEq, Encode, Decode, Eq)]
+pub enum DirectRequestStatus {
+    /// Direct request was successfully executed
+    #[codec(index = 0)]
+    Ok,
+    /// Trusted Call Status
+    /// Litentry: embed the top hash here - TODO - use generic type?
+    #[codec(index = 1)]
+    TrustedOperationStatus(TrustedOperationStatus, H256),
+    /// Direct request could not be executed
+    #[codec(index = 2)]
+    Error,
+}
+```
+
+### Using CLI
+
+There are two commands related to transaction signing:
+
+* request-direct-call-sign-bitcoin
+* request-direct-call-sign-ethereum
+
+They take single argument representing raw payload bytes to sign.
+
+#### Example usage
+
+```bash
+./bitacross-cli trusted -m 7ppBUcnjGir4szRHCG59p2dTnbtRwKRbLZPpR32ACjbK request-direct-call-sign-bitcoin 00
```

Comment:
Do we need `-m <shard>` here? 
I remember we had a PR that removed it (just try to make it as simple as possible)

Reply:
Recently it stopped working. CLI utils gets empty mrenclave so signature verification fails later.

```
Got mrenclave and shard [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] 0x0000000000000000000000000000000000000000000000000000000000000000
```